.example.env

```diff
@@ -1,6 +1,9 @@
-VITE_MAPBOX_TOKEN=REDACTED
-MAPBOX_STREETS_URL=https://api.mapbox.com/styles/v1/mapbox/streets-v11/tiles/{z}/{x}/{y}
+HUGGINGFACE_LOGIN_TOKEN=REDACTED
 MAPBOX_SATELLITE_URL=https://api.mapbox.com/styles/v1/mapbox/satellite-streets-v11/tiles/{z}/{x}/{y}
+MAPBOX_STREETS_URL=https://api.mapbox.com/styles/v1/mapbox/streets-v11/tiles/{z}/{x}/{y}
+VITE_ENV=DEV
+VITE_CONTACT_FORM_DEV=https://docs.google.com/forms/d/e/1FAIpQLScrhJY9ezqlKcpYermIImgGKYZ4bP2k1BRAOpuEay-tVXfvdQ/viewform?embedded=true
```

Comment:
(optional) change this to just be `VITE_CONTACT_FORM`. Let's not share _any_ of the google form links here
- make it `REDACTED`